compiler-cli/src/http.rs

```diff
@@ -42,3 +43,32 @@ impl gleam_core::io::HttpClient for HttpClient {
             .map_err(Error::http)
     }
 }
+
+fn init_client() -> reqwest::Client {
+    if let Some(cert) = get_certificate() {
+        return reqwest::Client::builder()
+            .add_root_certificate(cert)
+            .build()
+            .expect("Unable to initialize a reqwest HTTP client");
+    } else {
+        return reqwest::Client::new();
+    }
+}
+
+fn get_certificate() -> Option<reqwest::Certificate> {
+    match std::env::var(CERTS_ENV_VAR) {
```

Comment:
This can be written as `let certs_path = std::env::var(CERTS_ENV_VAR)?` to get rid of the nesting